PWGJE/Tasks/jetTaggerHFQA.cxx

```diff
@@ -1635,10 +1635,11 @@ using JetTaggerQAChargedDataJets = soa::Join<aod::ChargedJets, aod::ChargedJetCo
 using JetTaggerQAChargedMCDJets = soa::Join<aod::ChargedMCDetectorLevelJets, aod::ChargedMCDetectorLevelJetConstituents, aod::ChargedMCDetectorLevelJetFlavourDef>;
 using JetTaggerQAChargedMCPJets = soa::Join<aod::ChargedMCParticleLevelJets, aod::ChargedMCParticleLevelJetConstituents, aod::ChargedMCParticleLevelJetFlavourDef>;
 
-using JetTaggerhfQACharged = JetTaggerHFQA<JetTaggerQAChargedDataJets, aod::ChargedJetTags, JetTaggerQAChargedMCDJets, aod::ChargedMCDetectorLevelJetEventWeights, aod::ChargedMCDetectorLevelJetTags, JetTaggerQAChargedMCPJets, aod::ChargedMCParticleLevelJetEventWeights, aod::ChargedMCDetectorLevelJetsMatchedToChargedMCParticleLevelJets, aod::ChargedMCParticleLevelJetsMatchedToChargedMCDetectorLevelJets>;
+using JetTaggerhfQaCharged = JetTaggerHFQA<JetTaggerQAChargedDataJets, aod::ChargedJetTags, JetTaggerQAChargedMCDJets, aod::ChargedMCDetectorLevelJetEventWeights, aod::ChargedMCDetectorLevelJetTags, JetTaggerQAChargedMCPJets, aod::ChargedMCParticleLevelJetEventWeights, aod::ChargedMCDetectorLevelJetsMatchedToChargedMCParticleLevelJets, aod::ChargedMCParticleLevelJetsMatchedToChargedMCDetectorLevelJets>;
```

Comment:
It would be more readable to name the `struct` `JetTaggerHfQaCharged` and so that it's hyphenated as `jet-tagger-hf-qa-charged`.
I see that `JetTaggerHfQaCharged` is actually an instance of a `JetTaggerHFQA` template which makes me suspect that the resulting device name would be `jet-tagger-h-g-q-a` by default. Can you please check it when you run the workflow?

Reply:
I've tried to modified to remove taskName but it doesn't work.
errors occur below
```
adaptAnalysisTask<JetTaggerhfQaCharged>(cfgc)};
[ERROR] invalid workflow in /Users/hanseopark/alice/sw/BUILD/c5d64f62ec99e24d89636f0e2145bbbcb900277d/O2Physics/stage/bin/o2-          analysis-je-jet-taggerhf-qa: Cannot use any of ,;:"'$ as DataProcessor name
```

Reply:
> > I've tried to modified to remove taskName but it doesn't work. errors occur below
> > ```
> > adaptAnalysisTask<JetTaggerhfQaCharged>(cfgc)};
> > [ERROR] invalid workflow in /Users/hanseopark/alice/sw/BUILD/c5d64f62ec99e24d89636f0e2145bbbcb900277d/O2Physics/stage/bin/o2-          analysis-je-jet-taggerhf-qa: Cannot use any of ,;:"'$ as DataProcessor name
> > ```
> 
> That looks like invalid syntax. What was the exact line you tried to compile?

  ```
 WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   {
     return WorkflowSpec{
       adaptAnalysisTask<JetTaggerhfQaCharged>(cfgc)};
   }
```
it is compiled but does not working.

Reply:
OK. Thanks for trying. Looks like `TaskName` is needed because of the template. Please use `using JetTaggerHfQaCharged = ...` and
```c++
adaptAnalysisTask<JetTaggerHfQaCharged>(cfgc, TaskName{"jet-tagger-hf-qa-charged"})}; // o2-linter: disable=name/o2-task (templated struct)
```

Reply:
I understand then I've changed them.
If I change `"jet-tagger-hf-qa-charged"` I should change `"jet-taggerhf-data-charged"` to "jet-tagger-hf-data-charged" and `"jet-taggerhf-mcd-charged"` to` "jet-tagger-hf-mcd-charged"` in `TableProducer/jetTaggerHF.cxx` for uniformity. Do you think changing everything is for unity? 
And what do you think about the suggestion for the “no matching function” error that occurs when there are more than 100 configurations?